src/diff.jl

```diff
@@ -51,7 +51,7 @@ is_derivative(x) = iscall(x) ? operation(x) isa Differential : false
 Base.:*(D1, D2::Differential) = D1 ∘ D2
 Base.:*(D1::Differential, D2) = D1 ∘ D2
 Base.:*(D1::Differential, D2::Differential) = D1 ∘ D2
-Base.:^(D::Differential, n::Integer) = _repeat_apply(D, n)
+Base.:^(D::Differential, n::Integer) = iszero(n) ? D.x : _repeat_apply(D, n)
```

Comment:
```suggestion
Base.:^(D::Differential, n::Integer) = iszero(n) ? identity : _repeat_apply(D, n)
```

The zeroth derivative of a function is the function itself, right?

Reply:
My initial understanding was that `Differential(x)` takes the derivative of `x` (as you can see by my initial definition as `D.x`), but the docstring implies that it's just the operator `d/dx` - so `Differential(x)^n * f = d^nd/dx^n`, right? Implying `Differential(x)^0 = d^0/dx^0 = 1`. Since it gets used as a callable object, it should be `identity`.

Reply:
For some reason I thought I already committed my change to `identity` and your diff was changing `identity` back to `D.x`. Sorry for the confusion :-)